### ISSUE TITLE
no implicit conversion of Fixnum into String

### ISSUE THREAD

**Issue by fertobar** [google/signet#56]
Hi.
I'm using Rails 4, & ruby 2.1 and I'm getting this error in SOAP connection:
no implicit conversion of Fixnum into String

More details:
rails (4.1.0)
faraday (0.9.1)

    google-ads-common (0.9.8)
      httpi (~> 1.1.0)
      savon (~> 1.2.0)
      signet (~> 0.6.0)
    google-adwords-api (0.14.2)
      google-ads-common (~> 0.9.8)
    google-dfp-api (0.9.4)
      google-ads-common (~> 0.9.4)

I notice that similar code works in Rails 3.2.20,and fails with Rails 4.1 , 4.2, and similar Rails 4

https://github.com/google/signet/blob/master/lib/signet/oauth_2/client.rb Line 748:

```ruby
      def expires_at
        if @expires_at
          @expires_at
        elsif @issued_at && @expires_in
          return @issued_at + @expires_in #TODO: @issued_at is String, String + Fixnum fails. 
        else
          return nil
        end
      end
```

Related with issue: https://github.com/googleads/google-api-ads-ruby/issues/53#issuecomment-96950402

Any suggestion?
thanks

**Comment by cavneb** [comment]
:+1:

**Comment by bruzos** [comment]
Same problem when using google-ruby-client  in service Google::Apis::AnalyticsV3::AnalyticsService with Rails 4.2.2 and ruby 2.2.1. but not in rails console

**Comment by Gataniel** [comment]
Facing the same issue when using `google-adwords-api`. Of course better solution was to locate and fix the error there, but I failed. In case someone may need implemented @bruzos solution https://github.com/Gataniel/signet/tree/fix-oauth2-client-expired-at

**Comment by Gataniel** [comment]
This issue IMHO is on `google-adwords-api`. So you here is my PR fixing it https://github.com/Gataniel/google-api-ads-ruby/tree/fix-ads_common-saving-issued_at-as-a-string-instead-of-time

**Comment by sqrrrl** [comment]
Should be fixed in 0.7
